R/fit.R

```diff
@@ -334,27 +340,37 @@ CmdStanMCMC <- R6::R6Class(
       }
     },
 
-    draws = function(inc_warmup = FALSE) {
-      if (is.null(private$draws_)) {
-        private$read_csv_()
+    draws = function(inc_warmup = FALSE, parameters = NULL) {
```

Comment:
What do you think we should do with the name `parameters`? rstan, bayesplot, and other packages use `pars`. posterior uses `variables`. I think we should go with `pars` or `variables` instead of `parameters` but I'm not sure which.

Reply:
Also we will probably eventually want to allow turning on matching via regex or some other mechanism so that people can select parameters more easily. Right now if you have `vector[3] theta` in your Stan program you can't do `parameters="theta"`, you have to do `parameters=list("theta[1]", "theta[2]", "theta[3]")`. This doesn't necessarily have to be included in this PR though.

Reply:
@paul-buerkner Any opinion on `pars` vs `variables` for the argument name?

Reply:
Lets go with pars if anyone doesnt ming. That is what the rstan users are accustomed to. I dont really like it, params is at least to me a more standard abbreviation. But probably best to avoid confusion.

Reply:
I really would prefer `variables` to pars.  Just writing up my reasons why as part of CmdStan3 design doc now - because I don't know where else to discuss this.

Reply:
> I really would prefer variables to pars. Just writing up my reasons why as part of CmdStan3 design doc now - because I don't know where else to discuss this.

@mitzimorris Sorry I missed this comment. Are you still OK with what we talked about in the meeting today? That is, going with `pars` for now for consistency with with rstan/pystan/bayesplot, but then potentially deprecating and changing it at a later date (e.g. with CmdStan3)?